### ISSUE TITLE
KDE4 Plasma Applet Conflicts, Addons Will Not Install

### ISSUE THREAD

**Issue by MichaelTunnell** [Antergos/kde-setup#5]
### Issue:
I decided to install the icontasks applet to create a dock-like panel for my KDE setup but the package would not install. It turns out that if files already exist in the **/usr/share/apps/desktoptheme/default/** folder then the packages detect a conflict and refuse to install.

### Solution:
Rename the folders of the applets to, example, "icontasks-bak" and the applets will install fine.

### Conflicts List:
Once I found this conflict I started testing other addons that seem to have files in that folder and I found the following addons and their conflicts.

1. kdeplasma-addons-applets-bubblemon
1. kdeplasma-addons-applets-fifteenpuzzle
1. kdeplasma-addons-applets-icontasks
1. kdeplasma-addons-applets-rssnow
1. kdeplasma-addons-applets-weather
1. kdeplasma-addons-applets-weatherstation

### Conflicts Details:
**kdeplasma-addons-applets-bubblemon**
* kdeplasma-addons-applets-bubblemon: /usr/share/apps/desktoptheme/default/bubblemon/bubble.svg exists in filesystem

**kdeplasma-addons-applets-fifteenpuzzle**
* kdeplasma-addons-applets-fifteenpuzzle: /usr/share/apps/desktoptheme/default/fifteenPuzzle/blanksquare.svg exists in filesystem

**kdeplasma-addons-applets-icontasks**
* kdeplasma-addons-applets-icontasks: /usr/share/apps/desktoptheme/default/icontasks/badge.svgz exists in filesystem
* kdeplasma-addons-applets-icontasks: /usr/share/apps/desktoptheme/default/icontasks/dropindicators.svgz exists in filesystem
* kdeplasma-addons-applets-icontasks: /usr/share/apps/desktoptheme/default/icontasks/indicators.svgz exists in filesystem
* kdeplasma-addons-applets-icontasks: /usr/share/apps/desktoptheme/default/icontasks/launcherseparator.svgz exists in filesystem
* kdeplasma-addons-applets-icontasks: /usr/share/apps/desktoptheme/default/icontasks/progress.svgz exists in filesystem

**kdeplasma-addons-applets-rssnow**
* kdeplasma-addons-applets-rssnow: /usr/share/apps/desktoptheme/default/rssnow/arrows.svgz exists in filesystem
* kdeplasma-addons-applets-rssnow: /usr/share/apps/desktoptheme/default/rssnow/background.svgz exists in filesystem
* kdeplasma-addons-applets-rssnow: /usr/share/apps/desktoptheme/default/rssnow/rssnow.svgz exists in filesystem

**kdeplasma-addons-applets-weather**
* kdeplasma-addons-applets-weather: /usr/share/apps/desktoptheme/default/weather/wind-arrows.svgz exists in filesystem

**kdeplasma-addons-applets-weatherstation**
* kdeplasma-addons-applets-weatherstation: /usr/share/apps/desktoptheme/default/weatherstation/lcd2.svgz exists in filesystem
* kdeplasma-addons-applets-weatherstation: /usr/share/apps/desktoptheme/default/weatherstation/lcd_digits.svgz exists in filesystem
* kdeplasma-addons-applets-weatherstation: /usr/share/apps/desktoptheme/default/weatherstation/lcd_panel.svgz exists in filesystem
* kdeplasma-addons-applets-weatherstation: /usr/share/apps/desktoptheme/default/weatherstation/weather_icons.svgz exists in filesystem
* kdeplasma-addons-applets-weatherstation: /usr/share/apps/desktoptheme/default/weatherstation/wind_arrows.svgz exists in filesystem

**Comment by lots0logs** [comment]
In this case its safe to use pacman's `--force` argument.

**Comment by MichaelTunnell** [comment]
Yea I understand that, I just mean for future installs it should probably be addressed so there aren't any conflicts. I already took care of it on my end.

**Comment by lots0logs** [comment]
Right, I'm just not so sure this is an issue to be handled by us. It seems more like a packaging issue. I will  see what I can find out.
